### PR TITLE
Update to support delete Mongo command. Issue #210

### PR DESCRIPTION
**Pull request by coreyjv**

### CODE REVIEW

File: src/main/java/com/github/fakemongo/FongoConnection.java [hunk 1]
```diff
@@ -470,6 +472,35 @@ private int delete(DBCollection collection, WriteConcern writeConcern, List<Dele
...
+      final FongoDBCollection dbCollection = (FongoDBCollection) db.getCollection(command.get("delete").asString().getValue());
+      List<BsonValue> documentsToDelete = command.getArray("deletes").getValues();
+
+      int numDocsDeleted = 0;
+      for (BsonValue document : documentsToDelete) {
+        BsonDocument deletesDocument = document.asDocument();
+
+        DBObject deleteQuery = dbObject(deletesDocument.get("q").asDocument());
+
+        BsonInt32 limit = (BsonInt32) deletesDocument.getOrDefault("limit", new BsonInt32(-1));
```
Comment (twillouer) [inline src/main/java/com/github/fakemongo/FongoConnection.java]: are you sure this is a java 6 method ?

Comment (coreyjv) [inline src/main/java/com/github/fakemongo/FongoConnection.java]: I'll take a look later. Nice catch.

**Comment by coreyjv** [comment]
@twillouer --

It should be all set now.

**Comment by twillouer** [comment]
Thanks !

**Pull request merged**